config/docker/rustc-1.62.jinja2

```diff
@@ -0,0 +1,42 @@
+{% extends 'clang-15.jinja2' %}
+
+{% block packages %}
+{{ super() }}
+
+ENV RUST_VER=1.62.0
+ENV BINDGEN_VER=0.56.0
+
+ENV RUST_TRIPLE=rust-${RUST_VER}-x86_64-unknown-linux-gnu
+
+ENV CARGO_HOME=/usr/$RUST_TRIPLE/cargo
```

Comment:
Should these (and the above ones) be `ARG`?

Reply:
My pleasure!

Reply:
`PATH` for sure, that is why I didn't include it.

As for `CARGO_HOME`, is there a reason to not let it use the default at the home folder? It feels weird to put it inside `/usr`, since it is meant to be a writable cache.

Reply:
Are you sure Cargo home supports being shared across users?

Reply:
How did you verify it? i.e. note that it is not just that it works, it would need to be designed for it. For instance, if it uses lock files or similar, does it properly work for several users? Or, since the cache is writable, does it have security implications? etc.

Considering the default is per-user, unless the docs state it is supported, I would be very cautious about it.